src/com2ann.py

```diff
@@ -9,19 +9,19 @@
 of additional (non-type) comments. For function definitions, we might introduce
 some formatting modifications, if the original formatting was too tricky.
 """
```

Comment:
```suggestion
"""
from __future__ import annotations

```
And then please update `lint_python.yml` to use `pyupgrade --py38-plus` instead of `--py36-plus`.